src/industry_gui.cpp

```diff
@@ -1210,12 +1342,19 @@ class IndustryDirectoryWindow : public Window {
 
 			this->industries.shrink_to_fit();
 			this->industries.RebuildDone();
-			this->vscroll->SetCount((uint)this->industries.size()); // Update scrollbar as well.
 		}
 
-		if (!this->industries.Sort()) return;
 		IndustryDirectoryWindow::last_industry = nullptr; // Reset name sorter sort cache
-		this->SetWidgetDirty(WID_ID_INDUSTRY_LIST); // Set the modified widget dirty
+
+		auto filter = std::make_pair(this->cargo_filter[this->accepted_cargo_filter_criteria],
+			                         this->cargo_filter[this->produced_cargo_filter_criteria]);
```

Comment:
bad indentation here

Reply:
In my editor the two parts of the or expression are under each other and align nicely. Here the second one is skewed to right. Should I try to align them under each other, or the second line should goes to nearer to beginning of the line "at all cost"?

Reply:
This is because your tab width is different to github's :)
There's an extra tab on this line, which messes with the alignment. Turn that into spaces, and all should be well.